src/mail.ts

```diff
@@ -20,6 +20,6 @@ export interface EmailParams {
   readonly subject: string;
   readonly body: string;
   readonly contentType?: string;
-  readonly headers?: string;
+  readonly headers?: object;
```

Comment:
I usually try to avoid the `object` type, as it is very wide. So instead I suggest you change the type to `Record<string, string>`. What do you think?

```suggestion
  readonly headers?: Record<string, string>;
```